tox.ini

```diff
@@ -46,3 +46,19 @@ setenv =
 basepython=python3.7
 deps=flake8>=2.2.0
 commands=flake8 wagtail_localize
+
+
+[testenv:interactive]
+basepython = python3.9
+
+commands_pre =
+    python {toxinidir}/testmanage.py makemigrations
```

Comment:
I needed this mainly for making migrations, but it is also useful to quicky spin up a test environment without the need to use bakery demo